internal/decryption/encrypted_audit_log.go

```diff
@@ -27,26 +27,26 @@ type AuditRecordType string
 type AuditLogLineKeyStoreIdentifier struct {
 	Provider *keyproviders.KeyStoreProvider `json:"provider,omitempty"`
 	// localKey
-	Filename *string `json:"filename,omitempty"`
+	Filename string `json:"filename,omitempty"`
 	// kmip
-	UID            *string                         `json:"uniqueKeyID,omitempty"`
-	KMIPServerName []string                        `json:"kmipServerName,omitempty"`
-	KMIPPort       *int                            `json:"kmipPort,omitempty"`
-	KeyWrapMethod  *keyproviders.KMIPKeyWrapMethod `json:"keyWrapMethod,omitempty"`
+	UID            string                         `json:"uniqueKeyID,omitempty"`
```

Comment:
same for all these pointer removals, be sure zero or empty strings are not valid values or remove the `omitempty`